.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea
 spcs/examples/llama7b-chat-hf/snow/__pycache__
 spcs/examples/llama7b-chat-hf/spcs_spec.yaml
+.DS_Store
+user-metrics/prometheus/Dockerfile
```

Comment:
why are we adding prometheus dockerfile to gitignore?

Reply:
yeah, it should be `user-metrics/metrics-service.yml`